src/display/llemu.c

```diff
@@ -189,6 +186,11 @@ bool _lcd_clear_line(lv_obj_t* lcd_dummy, int16_t line) {
 	return true;
 }
 
+void _lcd_clear(lv_obj_t* lcd_dummy) {
+	for (size_t i = 0; i < LLEMU_LINES; i++)
+    	_lcd_clear_line(lcd_dummy, i);
```

Comment:
fix indentation in this function and add braces for the loop

Reply:
The indentation looks good to me. Not sure why it looks different on github.

Reply:
I think we should use tabs here